elliott/elliottlib/util.py

```diff
@@ -558,3 +558,17 @@ def log(msg):
         r = labels['release']
         all_payload_nvrs[component] = (v, r)
     return all_payload_nvrs
+
+
+def fix_summary_suffix(summary, summary_suffix):
+    # get summary of CVE tracker bug and update summary if needed
+    if '[' in summary and ']' in summary:
```

Comment:
what if there are multiple "[..]" .. in the summary ? 

i think we should look for a [openshift-*] pattern , and only replace that

if it is not found, then add the pattern

Reply:
Done